arkane/statmech.py

```diff
@@ -158,6 +158,9 @@ def hinderedRotor(scanLog, pivots, top, symmetry=None, fit='best'):
     """Read a hindered rotor directive, and return the attributes in a list"""
     return [scanLog, pivots, top, symmetry, fit]
 
+def hinderedRotor1DArray(angles, energies, pivots, top, symmetry=None, fit='best'):
+    """Read a hindered rotor PES profile, and return the attributes in a list"""
+    return [angles, energies, top, pivots, symmetry, fit]
```

Comment:
Is the order of `pivots` and `top` reversed on purpose between the input args and the output?

Reply:
Thanks for finding it out. It should be a mistake.

Reply:
Fixed